src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -546,12 +568,19 @@ export default class AircraftController {
     _createAircraftWithInitializationProps(initializationProps) {
         const aircraftModel = new AircraftModel(initializationProps);
         const isDeparture = initializationProps.category === 'departure';
+        const isArrival = initializationProps.category === 'arrival';
         const isAutoTower = GameController.getGameOption(GAME_OPTION_NAMES.TOWER_CONTROLLER) === 'SYSTEM';
+        const runwayCommands = initializationProps.commands;
 
         // triggering event bus rather than calling locally because multiple classes
         // are listening for the event and aircraft model
         this._eventBus.trigger(EVENT.ADD_AIRCRAFT, aircraftModel);
 
+        if (isArrival && runwayCommands && runwayCommands[aircraftModel.fms.arrivalRunwayModel.name]) {
+            const commandString = runwayCommands[aircraftModel.fms.arrivalRunwayModel.name];
+            this._runCommandOnNewAircraft(aircraftModel, commandString);
```

Comment:
missing new line between assignment and method call

Reply:
done